app/src/main/java/com/keylesspalace/tusky/components/preference/AccountPreferencesFragment.kt

```diff
@@ -194,6 +194,28 @@ class AccountPreferencesFragment : PreferenceFragmentCompat(), Injectable {
                     }
                 }
 
+                val activeAccount = accountManager.activeAccount
+                if (activeAccount != null) {
+                    listPreference {
+                        setTitle(R.string.pref_default_reply_privacy)
+                        setEntries(R.array.post_privacy_names)
+                        setEntryValues(R.array.post_privacy_values)
+                        key = PrefKeys.DEFAULT_REPLY_PRIVACY
+                        setSummaryProvider { entry }
+                        val visibility = activeAccount.defaultReplyPrivacy
```

Comment:
Something like this instead?

```suggestion
                        val visibility = accountManager.activeAccount?.defaultReplyPrivacy ?: accountManager.activeAccount?.defaultPostPrivacy ?: Status.Visibility.PUBLIC
```

That way you can remove the null check on line 198, and this will fallback through their default post privacy if this is not set, or public otherwise (consistent with how the other visibility preference is managed).

Reply:
?

Reply:
It sounded like we didn't want to implement this until it was implemented server-side so I haven't really been focusing on this pr